Homestead/readme.md

```diff
@@ -61,6 +61,7 @@ features:
 folders:
     - map: ~/Code
       to: /home/vagrant/Code
+      type: "nfs"
```

Comment:
👀  What was spooky about it that this resolved? 👻  (just curious)

Reply:
I'd been running into weird errors when running `composer install` that seem to boil down to VirtualBox [messing up file permissions](https://git.io/JeKcq). From what I can gather it's due to some sort of bug in VirtualBox's default folder syncing implementation that's went unresolved for a long time. 🤷‍♂️

Reply:
Aha! Ok, I was asking cause I was curious if likely related to the same Homestead/VirtualBox issue I ran into when re-setting up my personal computer a month ago! I found this issue which seems to still not be resolved... but the linked comment solution did it for me! I think when I tried `nfs` on my local it hadn't completely worked but maybe that was just a fluke.

https://github.com/laravel/homestead/issues/1240#issuecomment-555951201

Reply:
Oh wait! Your link goes to the same issue, hahahah 😂

Reply:
The `--prefer-source` seemed to get things going for me, but the `nfs` setting sounds like a more consistent fix.

Reply:
I hope! It's a really spooky issue, so who knows though. 👻